main.nf

```diff
@@ -800,6 +830,7 @@ process kraken2 {
     output:
     set val("kraken2"), val(name), file("results.krona") into kraken2_to_krona
     file("kraken2_report.txt")
+    val("${task.cpus}") into ch_kraken2_cpus
```

Comment:
Can't find where this is used `ch_kraken2_cpus`. Is that need for reproducibility as well?

Reply:
well spotted :)